src/web/lib/getCommentContext.ts

```diff
@@ -101,9 +91,15 @@ export const getCommentContext = async (
 ): Promise<CommentContextType> => {
 	const url = joinUrl([ajaxUrl, 'comment', commentId.toString(), 'context']);
 	const filters = initFiltersFromLocalStorage();
-	const params = buildParams(filters);
+	const paramsObj = buildParams(filters);
+
+	const params = Object.keys(paramsObj)
+		// @ts-ignore
+		// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+		.map((key) => `${key}=${paramsObj[key]}`)
```

Comment:
I think we want to move this refactor out so that we can verify it and maybe add some tests? Ideally we can work out a way too achieve this without having to use ts-ignore but lets see